modules/users/client/controllers/admin/user.client.controller.js

```diff
@@ -42,8 +44,10 @@
         $state.go('admin.user', {
           userId: user._id
         });
+        Notification.success('User updated successfully!');
       }, function (errorResponse) {
         vm.error = errorResponse.data.message;
+        Notification.error('User update error!');
```

Comment:
@sujeethk Do we still need to set `vm.error` in these cases? Or would it be better to add the response error as the body of the toast, and keep "User update error!" as the title?